src/R/aspect.R

```diff
@@ -0,0 +1,25 @@
+# @author Lauren Bolotin
+# @email lauren.bolotin@noaa.gov
+# @date  Aug 25, 2025
+
+# ########################### Slope ########################
+# Function computes aspect from a DEM to replace the hydrofabric aspect values
+
+aspect_function <- function(div_infile, dem_output_dir) {
+  div <- read_sf(div_infile, 'divides')
+
+  dem <- rast(glue("{dem_output_dir}/dem.tif"))
```

Comment:
can you use the corrected DEM here, i.e., `dem_corr`, the original DEM might have some NaNs or other issues